lib/flipper_app/authorize.rb

```diff
@@ -0,0 +1,50 @@
+require 'active_record'
+require 'exercism/user'
+
+module FlipperApp
+  class Authorize
+    def initialize(app, options = {})
+      @app = app
+    end
+
+    def call(env)
+      if authorized?(env)
+        @app.call(env)
+      else
+        forbidden
+      end
+    end
+
+    private
+
+    def authorized_usernames
+      %w(kytrinyx nilbus)
+    end
+
+    def authorized_github_ids
+      @authorized_github_ids ||=
+        User.where(username: authorized_usernames).map(&:github_id)
```

Comment:
We can use `pluck` here if we just want attributes. `User.where(username: authorized_usernames).pluck(:github_id)`